Source/Core/DolphinQt/Config/SettingsWindow.cpp

```diff
@@ -46,46 +46,32 @@ StackedSettingsWindow::StackedSettingsWindow(QWidget* parent) : QDialog{parent}
   m_navigation_list->setSizeAdjustPolicy(QAbstractScrollArea::AdjustToContents);
 
   // FYI: "base" is the window color on Windows and "alternate-base" is very high contrast on macOS.
-  const auto [list_background, list_hover_background] =
-#if defined(_WIN32)
-      std::pair("palette(alternate-base)", "palette(base)");
+  const auto* const list_background =
+#if !defined(__APPLE__)
+      "palette(alternate-base)";
 #else
-      std::pair("palette(base)", "palette(alternate-base)");
+      "palette(base)";
 #endif
 
   m_navigation_list->setStyleSheet(
       QString::fromUtf8(
           // Remove border around entire widget and adjust background color.
           "QListWidget { border: 0; background: %1; } "
-          "QListWidget::item { padding: %2; "
-#if defined(__linux__)
-          "} "
-#else
-          // For some reason this is needed to fix inconsistent item padding on Windows and macOS,
-          //  but it unfortunately also breaks item background color changes.
-          "border: 0; } "
-          // Restore selected item color.
+          // Note: padding-left is broken on Windows unless border is set, which breaks colors.
+          // see: https://bugreports.qt.io/browse/QTBUG-122698
+          "QListWidget::item { padding-top: %2; padding-bottom: %2; padding-right: %2; } "
```

Comment:
This _may_ be relying on undefined behavior. At least in CSS, not specifying units is considered an error (except when the value is 0).

```suggestion
          "QListWidget::item { padding-top: %2px; padding-bottom: %2px; } "
```